app/src/main/java/com/example/android/hilt/ui/MainActivity.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("DEPRECATION")
```

Comment:
Hmm not sure if we should add this here, can we remove the changes to this file?

Reply:
Sure, it will be removed.